geonode/static/geonode/css/base.css

```diff
@@ -3061,10 +3061,15 @@ select[multiple].input-lg {
 .form-horizontal .checkbox {
   min-height: 29px;
 }
+
```

Comment:
Any change made to "base.css" must go to "base.less" first, otherwise, it will get lost at next "grunt" compile.

Reply:
Have moved all the css additions to the base.less file. 

There was one part of base.css which was commented out, however this does not seem to be present in the base.less file so assuming nothing needs to be done for this.